src/transformers/models/t5/convert_t5x_checkpoint_to_pytorch.py

```diff
@@ -217,5 +223,8 @@ def convert_t5x_checkpoint_to_pytorch(t5x_checkpoint_path, config_file, pytorch_
     parser.add_argument(
         "--pytorch_dump_path", default=None, type=str, required=True, help="Path to the output PyTorch model."
     )
+    parser.add_argument(
+        "--is_encoder_only", action='store_true', help="Check if the model is encoder-decoder model"
+    )
```

Comment:
Let's use a `default`, this parameter is a bit confusing IMO

Reply:
I get confused a lot with this too 😂